lib/process-services-cloud/src/lib/form/components/widgets/dropdown/dropdown-cloud.widget.ts

```diff
@@ -310,4 +310,8 @@
             return this.field?.restUrl;
         }
     }
+
+    showRequiredMessage(): boolean {
+        return (this.isInvalidFieldRequired() || this.field.value === 'empty') && this.isTouched() && !this.isRestApiFailed;
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Alfresco/alfresco-ng2-components/pullRequest?prid=8980729)
```suggestion
        return (this.isInvalidFieldRequired() || this.field.value === "empty") && this.isTouched() && !this.isRestApiFailed;
```